### ISSUE TITLE
Kirchhoff formulation fails at loading Intel MKL libraries

### ISSUE THREAD

**Issue by BartelsJ** [openego/eTraGo#177]
When switching from angles to the kirchhoff formulation at network.lopf (several scenarios and data set versions),  I receive a 'Kernel died, restarting' remark from spyder and following information at my terminal:
**Intel MKL FATAL ERROR: Cannot load libmkl_avx512.so or libmkl_def.so.**
(mkl is installed at version 2017.0.3)

As suggested [here](https://github.com/BVLC/caffe/issues/3884), the example command line 
'python -c 'import sklearn.linear_model.tests.test_randomized_l1'
runs withour errors. Hence, anaconda seems to run correctly. 

Any experiences on that error? Are you using a different mkl version or does it run without problems?

**Comment by wolfbunke** [comment]
Does this issue needs a Milestone?

**Comment by BartelsJ** [comment]
No. I guess this is still an open issue but the functionality of eTraGo is not endangered.
